.github/workflows/build_and_test.yml

```
@@ -6,45 +6,174 @@ on:
+        else
+          COMMIT_ID=${{ github.event.head_commit.id }}
+          COMMIT_AUTHOR_EMAIL=${{ github.event.head_commit.author.email }}
+          COMMIT_AUTHOR_NAME=${{ github.event.head_commit.author.name }}
+        fi
+
+        echo "COMMIT_ID=${COMMIT_ID}" >> $GITHUB_OUTPUT
+        echo "COMMIT_AUTHOR_EMAIL=${COMMIT_AUTHOR_EMAIL}" >> $GITHUB_OUTPUT
+        echo "COMMIT_AUTHOR_NAME=${COMMIT_AUTHOR_NAME}" >> $GITHUB_OUTPUT
+
+    - name: Login to Docker Container Registry (no cache)
+      if: |
```

actionlint (1.7.7)

property "check_existing_image" is not defined in object type {check_commit_message: {conclusion: string; outcome: string; outputs: {string => string}}; determine_commit_id_and_email_author: {conclusion: string; outcome: string; outputs: {string => string}}; get_job_id: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)